### ISSUE TITLE
It would be convenient if a user could open up the Destination directory/folder (DestDir) in File Explorer, similar to what can be done with the Source directory/folder

### ISSUE THREAD

**Issue by ZPNRG** [FastCopyLab/FastCopy#235]
It would be convenient if a user could open up the Destination directory/folder (DestDir) in File Explorer, similar to what can be done with the Source directory/folder.

A simple solution would probably be to provide a similar drop-down menu arrow for DestDir as is provided for Source.  This drop-down menu could provide a short list of recently used DestDir directories/folders and also provide the same (Open Explorer) option which would open the current director/folder in the DestDir field.

After a Copy, I often want to go to the DestDir and make sure everything looks fine, run my hash checksums utility to verify my pre-existing hashes, etc.  This option to (Open Explorer) would allow me to quickly open the DestDir folder.

**Comment by shirouzu** [comment]
Since the DestDir field is a combo box, the history is displayed by pressing the button on the far right.
Also, in v5.4.1, the DestDir contents can be opened in Explorer by holding down the Shift key and pressing the DestDir button.
![image](https://github.com/FastCopyLab/FastCopy/assets/1759911/8898b019-db48-4f82-bcf9-db72f3adf941)

**Comment by ZPNRG** [comment]
@shirouzu  I'm so sorry I had somehow not noticed the drop-down arrow to the right of the DestDir field.  Maybe I had noticed and used it before, but if I had, I obviously had a mental flub the day I reported this issue.  I also had not noticed that if I hovered my mouse over the DestDir button, it says "Shift key to open File Explorer".  So, I'm glad that feature is there.

However, for consistency, it might be best to duplicate the functionality of Source button and that area for DestDir, putting the drop-down arrow also on the left side below the DestDir button (for consistency) as well as also adding the (Open Explorer) option to the drop-down menu as well.  This way, both the Source and DestDir button can be clicked on with Shift to open File/Windows Explorer or, if the user doesn't notice that option by hovering their mouse over the Source/DestDir button, they would still see the option in the drop-down menu under each button.
